sources/settings-extensions/ntp/src/lib.rs

```diff
@@ -65,6 +66,7 @@ mod test {
             NtpSettingsV1::generate(None, None),
             Ok(GenerateResult::Complete(NtpSettingsV1 {
                 time_servers: None
+                options: None
```

Comment:
need commas after each of these items:

```suggestion
                time_servers: None,
                options: None,
```